src/unix/pipe.c

```diff
@@ -52,7 +52,7 @@ int uv_pipe_bind(uv_pipe_t* handle, const char* name) {
   name_len = strlen(name);
 
   if (name_len > sizeof(saddr.sun_path) - 1)
-    return -ENAMETOOLONG;
+    return UV_ENAMETOOLONG;
```

Comment:
This should probably be a separate commit/PR since it's not really related to Haiku support.